source/vocab/files-packages-representations.ttl

```diff
@@ -119,3 +119,13 @@ prefix : <https://id.kb.se/vocab/>
 :height a owl:DatatypeProperty ;
     sdo:domainIncludes :MediaObject ;
     owl:equivalentProperty sdo:height .
+
+:startTime a owl:DatatypeProperty ;
+    sdo:domainIncludes :MediaObject ;  #for event and bitstream?
+    owl:equivalentProperty sdo:startTime ;
+    rdfs:label "start time"@en, "starttid"@sv.
+
+:endTime a owl:DatatypeProperty ;
+    sdo:domainIncludes :MediaObject ; #for event and bitstream?
+    owl:equivalentProperty sdo:endTime ;
+    rdfs:label "end time"@en, "sluttid"@sv.
```

Comment:
Given that these apply to `MediaObject` in general, we might want to move them to details.ttl (or conversely move `MediaObject` with details here if we have any).